gammapy/utils/metadata.py

```diff
@@ -191,6 +191,29 @@ def validate_time(cls, v):
         else:
             return v
 
+    def to_header(self, format="gadf"):
```

Comment:
This should not be in this PR. There might have been a problem when rebasing.

Reply:
Well, it has been removed with #4886 . You are introducing a former version of it...

Reply:
Ah apologies, I was looking at the wrong part of the code. I will rebase again

Reply:
Had to fix it manually as it was a commit that I had made, should be all good now